tasks/main.yml

```diff
@@ -68,8 +75,6 @@
       dest: "{{ workdir }}/openssl-1.0.cnf"
     with_items:
       - openssl-1.0.cnf.j2
-      - x509-types/kerberos-kdc
-      - x509-types/kerberos-user
```

Comment:
These two lines have no effect as `{{ item }}` isn't used anywhere in this task. As I'm not sure what to fix here, I removed it.

Reply:
I went and tracked down the single installation where those two files are used - and indeed, this was never deployed properly. Good catch! I'll fix this up in a subsequent PR.